src/app/register/components/form-visibility/form-visibility.component.html

```diff
@@ -0,0 +1,48 @@
+<ng-container [formGroup]="form">
+  <h3 class="mat-subheading-2" i18n="@@register.visibility">
+    Visibility settings
+  </h3>
+  <p>
+    <ng-container i18n="@@register.visibilityDescription1">
+      ORCID iD connects with your ORCID record that can contain links to your
+      research activities, affiliations, awards, other versions of your name,
+      and more. You control this content and who can see it.
+    </ng-container>
+    <a
+      href="https://support.orcid.org/hc/articles/360006897614"
+      target="_blank"
+      rel="noopener"
+      i18n="@@register.visibilityDescription2"
+      >More information on visibility settings.</a
+    >
+  </p>
+  <p i18n="@@register.visibilityDescription3">
+    By default, who should be able to see information added to your ORCID
+    Record?
+  </p>
+  <p>
+    <mat-radio-group
+      formControlName="activitiesVisibilityDefault"
+      color="primary"
+    >
+      <mat-radio-button
```

Comment:
@leomendoza123 Could you add a margin between the options they are to close to each other and the icons are missing

Reply:
Perfect added the margins, and we have a card for the icons https://trello.com/c/Q7sMADQg/6744-register-add-visibility-icons

Reply:
I have added the margin, and there is a card for the icons since those need to be defined by Dan https://trello.com/c/Q7sMADQg/6744-register-add-visibility-icons